src/main/java/com/sda/zdjavapol107/memorki/Main.java

```diff
@@ -0,0 +1,149 @@
+package com.sda.zdjavapol107.memorki;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
```

Comment:
poczatek jest dobry, ogolnie na pewno zrob jakas klase ktora bedzie zarzacac boardem :) bo bedziesz tam musial oznaczac ktore pary juz zostaly odkryte :D mozesz to zrobic na pare spoosbow albo jakas tablica pomocnicza albo rozszesz sobie typ ktory przechowujesz w boardzie, nie String tylko jakis wlasny obiekt ktory moze trzymac wiecej informacji :)